### DISCUSSION TITLE
Does this latest ATD support cloud automation (Headspin)

### DISCUSSION THREAD

**Discussion by SushanthN** [AppiumTestDistribution/AppiumTestDistribution#1087] [General]
@saikrishna321 Currently i can see even if we are running in cloud, we are starting appium server at client end. Also in Capability builder class we are not setting the udid of devices.

**Comment by saikrishna321** [comment]
No, we don’t support Headspin. Happy to review PR. Changes need to be done in Appium-device-farm plugin.
